### PR TITLE
Module 1: Collaboraton to add Contact for on Portfolio

### PR DESCRIPTION
**Pull request by Shedrack-Sunday**
Features Added:
1. HTML contact form and related links
2. CSS Styling of the contact form

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -330,3 +334,94 @@ a {
   display: flex;
   flex: 1;
 }
+
+.div-contact {
+  background-image: url("images/Frame.png");
+  background-position: bottom right;
+  background-repeat: no-repeat;
+}
```
Comment (Nemwel-Boniface) [inline style.css:342]: - [ ] Kindly note that according to the project requirements, you are required to use Flexbox to align the elements of this section. Kindly use Flexbox for this section.

image

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @Shedrack-Sunday            :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Great job working on your portfolio so far! :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Shedrack-Sunday) [reply style.css:342]: I used flexbox for all spaces necessary. This one isn't there. Find the instructions below.

""Make sure that:

Name field accepts up to 30 chars.
Email field accepts only emails.
Message field accepts up to 500 chars.
All fields have placeholder text.
All fields have the correct type.
A user cannot submit the form if one or more of the fields is empty.
Collect data by using the [Formspree service](https://formspree.io/html).
In order to lay out the elements on the page you should use Flexbox.
You don't need to implement any functionality that requires JavaScript.
You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
Implement the button interactions (enable, hover, etc.).""""

Comment (Shedrack-Sunday) [reply style.css:342]: `contact-form ` which is the child class and contains the content is displaced in flexbox. The requirement has been met.

**Review by danmainah — APPROVED** [review]
Hi @Shedrack-Sunday , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**